lib/net.c

```diff
@@ -838,10 +838,43 @@ check_port_mtu(struct gatekeeper_if *iface, unsigned int port_idx,
 	return 0;
 }
 
+static int
+check_port_cksum(struct gatekeeper_if *iface, unsigned int port_idx,
+	const struct rte_eth_dev_info *dev_info,
+	struct rte_eth_conf *port_conf, bool gatekeeper_server)
+{
+	if (ipv4_if_configured(iface) &&
+			(port_conf->txmode.offloads &
```

Comment:
Do we need to check this condition? It seems that if ``ipv4_if_configured()`` returns true, this flag will be always set before.

Reply:
I agree with @mengxiang0811. We shouldn't care with the test that leads to adding `DEV_TX_OFFLOAD_IPV4_CKSUM` to `port_conf->txmode.offloads` in `check_port_cksum()`, that is the job of `check_port_offloads()`. So we can safely remove `ipv4_if_configured(iface)` off of the test.

Reply:
Hi @cjdoucette, I see that you addressed the issue in the third patch. the fixes became confusing there. Could you split the third patch and move the fixes that @mengxiang0811 is talking about to the second patch?